functions/go/_template/README.md

```diff
@@ -6,23 +6,39 @@
 
 Explain what this function does in one or two sentences.
 
+Describe why the user should care about this function.
```

Comment:
It will be good to include the link to style guide for people to follow.

Reply:
+1